projects/example_project/README.md

```diff
@@ -0,0 +1,93 @@
+This is an example README for community `projects/` with full comments on [README_template.md](README_template.md). If you wish to submit your project to our main repository, then all the fields in [README_template.md](README_template.md) are mandatory for others to understand what you have achieved in this implementation. Should you still feel unclear about details, feel free to read our [contribution guide](https://mmocr.readthedocs.io/en/dev-1.x/notes/contribution_guide.html) or approach us in [Discussions](https://github.com/open-mmlab/mmocr/discussions) for help.
```

Comment:
Should have a big title indicating the algorithm, similar to those README.md in configs/